src/NUnitFramework/framework/Api/FrameworkController.cs

```diff
@@ -174,7 +177,15 @@ private void Initialize(string assemblyPath, IDictionary settings)
         /// <summary>
         /// Gets a dictionary of settings for the FrameworkController
         /// </summary>
-        public IDictionary Settings { get; private set; }
+        public IDictionary Settings
+        {
+            get { return (IDictionary)InternalSettings; }
+        }
+
+        /// <summary>
+        /// Gets a dictionary of settings for the FrameworkController
+        /// </summary>
+        internal IDictionary<string, object> InternalSettings { get; private set; }
```

Comment:
@CharliePoole made it very clear that the actual implementation defined the "published API", not any documentation.

I asked the question multiple times if this includes the properties of the public classes, as well as all of the public methods.  The implication of all replies to those was that if it is implemented and public, then it is part of the "published API" for those classes which are defined as being part of the API.

Now, something different is being said.  Someone needs to put a stake in the ground on this issue.

I did not want to have to do this InternalSettings things, but I felt compelled to by the statements that @CharliePoole  made on the "published API" issue.